.github/workflows/release-azure-webapp-v1.yaml

```diff
@@ -0,0 +1,268 @@
+      use_private_agent:
+        description: Use a private agent to deploy the built artifact.
+        type: boolean
+        required: false
+        default: true
+      use_labels:
+        description: Use labels to start the right environment's GitHub runner. If use_labels is true, also use_private_agent must be set to true
+        type: boolean
+        required: false
+        default: false
+      override_labels:
+        description: Needed for special cases where the environment alone is not sufficient as a distinguishing label
+        type: string
+        required: false
+        default: ""
+
+concurrency:
+  group: ${{ github.workflow }}-cd
+  cancel-in-progress: true
+
+env:
+  BUNDLE_NAME: ${{ inputs.workspace_name }}
+  RESOURCE_GROUP_NAME: ${{ inputs.resource_group_name }}
+  WEB_APP_NAME: ${{ inputs.web_app_name }}
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    env:
+      WORKSPACE_NAME: ${{ inputs.workspace_name }}
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Setup Node.js
+        id: node-setup
+        uses: pagopa/dx/.github/actions/node-setup@main
+
+      - name: Install dependencies (pnpm)
+        if: ${{ steps.node-setup.outputs.package-manager == 'pnpm' }}
+        run: pnpm --filter $WORKSPACE_NAME install
+
+      - name: Install dependencies (yarn)
+        if: ${{ steps.node-setup.outputs.package-manager == 'yarn' }}
+        run: yarn install --immutable
+
+      - name: Build
+        run: npx turbo build --filter $WORKSPACE_NAME
+
+      - name: Get workspace path
+        id: get-workspace-path
+        run: |
+          WORKSPACE_PATH=$(npx turbo ls --output=json |
+            jq -sr --arg NAME "$WORKSPACE_NAME" '.[].packages.items[] | select(.name == $NAME).path')
+          echo "path=$WORKSPACE_PATH" >> $GITHUB_OUTPUT
+
+      - name: Make artifact
+        id: make-artifact
+        env:
+          YARN_NODE_LINKER: node-modules
+          YARN_NM_HOISTING_LIMITS: workspaces
+          PACKAGE_MANAGER: ${{ steps.node-setup.outputs.package-manager }}
+        working-directory: ${{ steps.get-workspace-path.outputs.path }}
+        run: |
+          if grep -rq --include='next.config.*' 'output: "standalone"' .; then
+            echo "::debug:: the workspace contains a Next.js standalone app"
+            mv .next/static .next/standalone/.next
+            cd .next/standalone && zip -r $BUNDLE_NAME.zip .
+            echo "artifact-path=$(realpath $BUNDLE_NAME.zip)" >> $GITHUB_OUTPUT
+            exit 0
+          fi
+
+          if [ "$PACKAGE_MANAGER" = "pnpm" ]; then
+            pnpm --filter $WORKSPACE_NAME deploy --prod bundle
+            cd bundle && zip -r $BUNDLE_NAME.zip .
+            echo "artifact-path=$(realpath $BUNDLE_NAME.zip)" >> $GITHUB_OUTPUT
+            exit 0
+          fi
+
+          yarn config set nodeLinker node-modules
```

Comment:
suggestion: I'd add i`f [ "$PACKAGE_MANAGER" = "yarn" ];` or use a switch case in order to make this code more explicit, early returns are not easy to spot and reason about